nextjs-bi-directional/app1/pages/index.js

```diff
@@ -1,28 +1,89 @@
 import Head from "next/head";
+import Link from "next/link";
 import GreetingAppOne from "../components/GreetingAppOne";
-import { dependencies } from "../package.json";
 
-const RemoteComponent = ({ scope, module, ...props }) => {
-  if (!global[scope]) {
+const useDynamicScript = (url) => {
```

Comment:
`useDynamicScript` and `RemoteComponent` fixed the issue I was mentioning in this comment https://github.com/module-federation/module-federation-examples/pull/217#issue-454815809 💯

Reply:
I still don't understand why it fixed it though.

Before I was loading both remote scripts in the head of the Next.js app, once I switched to dynamic loading (`useDynamicSCript`) things started working. I'd like to understand why that happened 😅